scala.wake

```diff
@@ -23,26 +31,7 @@ def fetchCoursier Unit =
   | runJob
   | getJobOutput
 
-def myMakeStatePath x =
-  makePlan ("<makeStatePath>", x, Nil) Nil
-  | setPlanKeep      False
-  | setPlanEcho      Verbose
-  | setPlanFnOutputs (\_ x, Nil)
-  | runJobWith       virtualRunner
-  | getJobOutput
-
-# Maintain compatibility with Wit 0.12 and below
-# Wit versions <= 0.12 have a Scala plugin that fetches Coursier, Bloop, and
-# all dependencies specified in ivydependencies.json files at the root of wit
-# packages (where the wit-manifest.json) resides.
-# To maintain backwards compatibility, accept that these dependencies may have
-# already been fetched, use virtual jobs to own the files if they already exist
-# rather than fetching them ourselves.
-def coursierBin =
-  match (files bloopInstall `.*coursier.*`)
-    bin, Nil = myMakeStatePath bin
-    Nil      = fetchCoursier Unit
-    l        = makeBadPath (makeError "Multiple coursier executables found! {catWith ", " l}")
+def coursierBin = fetchCoursier Unit
```

Comment:
I think this is just going to force this to execute unconditionally on any Wake invocation, since this is at the top level with no arguments, and therefore this defeats the purpose of putting the `Unit` parameter in the definition of `fetchCoursier`. You should probably instead just replace wherever `coursierBin` is being used with `fetchCoursier Unit`.